### ISSUE TITLE
Removed files on Bedrock not in sync with those in www-l10n repo

### ISSUE THREAD

**Issue by peiying2** [mozilla/bedrock#14507]
## Description

There has been a few instances where files removed on Bedrock and the change don't trigger an update PR on www-l10n. Sometimes several changes in Bedrock trigger several commits on www-l10n that conflict between the PRs.

1). The most [recent removal change](https://github.com/mozilla/bedrock/commit/b8b1bb42321eaba134616ed55ecd6e903d2593ff) were the files under en/firefox/browsers/compare but there was no PR triggered  on the www-l10n repo, making them out of sync.  
Bedrock: https://github.com/mozilla/bedrock/tree/main/l10n/en/firefox/browsers
www-l10n: https://github.com/mozilla-l10n/www-l10n/tree/master/en/firefox/browsers

Craig tried to trigger it to work, and but it came out empty. He commented [here](https://github.com/mozilla-l10n/www-l10n/pull/394#issuecomment-2062592469). (The PR picked up other commits thus we merged it).

2). [This PR](https://github.com/mozilla-l10n/www-l10n/pull/384) on www-l10n saw some conflicting commits, files removed in the prior PR were reintroduced through a new commit.

### Expected result

- Identify the root cause of PRs that cancel out each other
- Find an interim solution to remove files under `en/firefox/browsers/compare`
- Remind dev to check that anytime they remove localizable files on Bedrock, there is an updated PR on the www-l10n repo. This change will remove `en` files. 
- The l10n team will be responsible to remove the corresponding localized files including `en-*` files.

@stevejalim

**Comment by janbrasna** [comment]
intermittent workflow fails for opening l10n PRs: https://github.com/mozilla/bedrock/actions/runs/8888752689/job/24406045061#step:3:35

```
docker: Error response from daemon: manifest for mozmeao/bedrock:9553c3dc9a767d46d6e464d2cfb84ea8a426da7b not found: manifest unknown: manifest unknown.
See 'docker run --help'.
Error: Process completed with exit code 125.
```

**Comment by stevejalim** [comment]
@janbrasna Yeah these are kind of race conditions where a new image hasn't yet hit the public repository/registry but a run triggered using that image has been started. This is partly why we run the sync every 3 hours, just in case
